view.js

```diff
@@ -0,0 +1,32 @@
+const view = () => {
+
+    
+    const render = (posts) => {
+        let postContainer;
+        let commentContainer;
+        $(`#posts`).empty()
+        for(let post of posts){
```

Comment:
You should split this up here into separate functions. I would say at least 2 - 3.